src/SFA.DAS.EmployerIncentives.Functions.LegalEntities/HandleBlockPaymentsRequest.cs

```diff
@@ -23,19 +23,22 @@ public HandleBlockPaymentsRequest(IBlockPaymentsService blockPaymentsService)
         }
 
         [FunctionName("HttpTriggerHandleBlockPaymentsRequest")]
-        public async Task<IActionResult> RunHttp([HttpTrigger(AuthorizationLevel.Function, "POST", Route = "block-payments")] HttpRequestMessage request)
+        public async Task<IActionResult> RunHttp(
+            [HttpTrigger(AuthorizationLevel.Function, "POST", Route = "block-payments")]
+            HttpRequestMessage request)
         {
             try
             {
                 var blockPaymentsRequest =
-                    JsonConvert.DeserializeObject<BlockAccountLegalEntityForPaymentsRequest>(await request.Content.ReadAsStringAsync());
+                    JsonConvert.DeserializeObject<List<BlockAccountLegalEntityForPaymentsRequest>>(
+                        await request.Content.ReadAsStringAsync());
                 await _blockPaymentsService.BlockAccountLegalEntitiesForPayments(blockPaymentsRequest);
             }
             catch (ArgumentException ex)
             {
-                return new ContentResult()
+                return new ContentResult
                 {
-                    StatusCode = (int) HttpStatusCode.BadRequest,
+                    StatusCode = (int)HttpStatusCode.BadRequest,
                     ContentType = "application/json",
                     Content = JsonConvert.SerializeObject(new
```

Comment:
This example JSON should be an array/list to reflect the changes to the interface

Reply:
This has been updated now